### DISCUSSION TITLE
I can't get carousel working with images

### DISCUSSION THREAD

**Discussion by thepurpleblob** [quasarframework/quasar#17532] [CLI - SSR mode]
I want to upload images into either my assets directory or my public directory to work with a carousel. I've not found any way to do it that doesn't result in a 404 error. There are no examples of doing this in the documentation (all images are external and have full URLs). Is this possible, if so how?

**Comment by ivanjaros** [comment]
https://quasar.dev/quasar-cli-vite/handling-assets

You either have to put asset path into template(with the ~ prefix) in order for the file to be detected and packaged during the build process, or you have to simply import the file, like you would a .js or vue template and then use it as computed/programmatic value to render.

Otherwise the assets will not get compiled.

**Comment by thepurpleblob** [comment]
Thanks - I wasn't aware of importing images. Found the docs and worked it out.
